draft-btw-add-home.txt

```diff
@@ -1205,34 +1205,18 @@ Internet-Draft       Encrypted DNS in Home Networks        November 2020
    Hosts serviced by legacy CPEs that can't be upgraded to support the
    options defined in Section 4 won't be able to learn the encrypted DNS
    server hosted by the ISP, in particular.  If the ADN is not
-   discovered using DHCP/RA, such hosts will have to fallback to use the
-   special-use domain name defined in [I-D.pp-add-resinfo] to discover
-   the encrypted DNS server and to retrieve the list of supported DoH
-   services using the RESINFO RRtype [I-D.pp-add-resinfo].
-
-   If the Do53 and Encrypted DNS servers are hosted on different public
-   IP addresses, the DNS server certificate can include the IP address
-   of the Do53 server in the subjectAltName extension.  After validating
-   the server certificate, the client can retrieve the IP address in the
-   certificate to identify that both Do53 and Encrypted DNS servers are
-   operated by the same entity.
-
-   If the Do53 and Encrypted DNS server are hosted on a private IP
-   address, they should use the same private IP address to prove to the
-   client they are operated by the same entity.
-
-   The DHCP/RA option to discover ADN takes precedence over special-use
-   domain name since the special-use domain name is susceptible to both
-   internal and external attacks whereas DHCP/RA is only vulnerable to
-   internal attacks.
-
-
-
-
-Boucadair, et al.          Expires May 6, 2021                 [Page 22]
-
-Internet-Draft       Encrypted DNS in Home Networks        November 2020
-
+   discovered using DHCP/RA, such hosts will have to fallback to use
+   DEER as defined in [I-D.pauly-add-deer] to discover the encrypted DNS
+   server and to retrieve the list of supported DoH services using the
+   SVCB RRtype [I-D.schwartz-svcb-dns] without verifying the hostname of
+   discovered templates with the ADN.  Other guidance in DEER relating
+   to resolver verification must be followed in this case.
+
+   The DHCP/RA option to discover DoH templates from DHCP or RA options
+   (should the WG pursue that approach pending feedback) takes
+   precedence over DEER since it uses unencrypted DNS to an external DNS
```

Comment:
"it" is confusing here, (it naturally refers to the DHCP/RA option, not DEER), so I suggest to replace it with "DEER".

Reply:
Good catch, thank you! (done)